### ISSUE TITLE
Custom default loading screen not working

### ISSUE THREAD

**Issue by Teskocf** [Gaerisson/Gmod-Custom-Menu#13]
https://i.imgur.com/8H0DPbP.png

**Comment by RyanTheTechMan** [comment]
This is a beautiful menu but yeah, it's a deal-breaker.
As soon as this is fixed I will gladly recommend everyone to use this menu.

I am unable to see any loading screen, even the one that a server normally has. ![img](https://i.gyazo.com/00d60169888031ecc7c5bda964099911.png)

**Comment by RyanTheTechMan** [comment]
Confused about this one... It's working completely fine now. May have been a GMod issue?
Strange!
(The original issue is not fixed, though custom loading screens work fine now 👍 )

**Comment by Gaerisson** [comment]
Yeah about the loading screen it works sometimes and sometimes not, idk why, but I still need to update the menu, I just need some times, there is alots of gmod update that break the menu :/

**Comment by Gaerisson** [comment]
About the blank screen issue, It is not appear with my game, but I think, it's the "nomusic.lua" module wich is supposed to cut the music of loading screen, maybe you joined a server that's using a fullscreen yt video for loading screen ?
and about the issue with loading screen with no informations, I will take a look to redo the page properlly.
